src/basic/missing.h

```diff
@@ -60,14 +60,6 @@
 #  define CIFS_MAGIC_NUMBER 0xFF534D42
 #endif
 
-#ifndef DRM_IOCTL_SET_MASTER
-#  define DRM_IOCTL_SET_MASTER _IO('d', 0x1e)
-#endif
-
-#ifndef DRM_IOCTL_DROP_MASTER
-#  define DRM_IOCTL_DROP_MASTER _IO('d', 0x1f)
-#endif
```

Comment:
Isn't this still used in logind?

Reply:
These are provided by kernel earlier than 3.11 in drm/drm.h. So, logind now includes drm/drm.h.